src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/contactperson/ContactPersonMappingResource.kt

```diff
@@ -24,7 +26,7 @@ import uk.gov.justice.hmpps.kotlin.common.ErrorResponse
 @Validated
 @PreAuthorize("hasRole('NOMIS_CONTACTPERSONS')")
 @RequestMapping("/mapping/contact-person", produces = [MediaType.APPLICATION_JSON_VALUE])
-class ContactPersonResource(private val service: ContactPersonService) {
+class ContactPersonMappingResource(private val service: ContactPersonService) {
```

Comment:
corrected name for consistency - though I guess we could remove the word "mapping" every where if we ever wanted to reduce noisy names